.github/workflows/ci.yaml

```diff
@@ -48,4 +48,7 @@ jobs:
       run: make list
 
     - name: Run CDK Synth
-      run: make synth
\ No newline at end of file
+      run: make synth
+    - run: npm publish
```

Comment:
Can't publish here yet. Need to run e2e tests before that. We can introduce something like RC publishing or dev preview, but not releases.